src/main/java/org/jabref/model/database/BibDatabase.java

```diff
@@ -242,29 +242,42 @@ private synchronized void insertEntries(List<BibEntry> newEntries, EntryEventSou
         }
     }
 
+    public synchronized void removeEntry(BibEntry bibEntry) {
+        removeEntries(Collections.singletonList(bibEntry));
+    }
+
+    public synchronized void removeEntry(BibEntry bibEntry, EntriesEventSource eventSource) {
+        removeEntries(Collections.singletonList(bibEntry), eventSource);
+    }
+
     /**
-     * Removes the given entry.
-     * The Entry is removed based on the id {@link BibEntry#id}
-     * @param toBeDeleted Entry to delete
+     * Removes the given entries.
+     * The entries removed based on the id {@link BibEntry#id}
+     * @param toBeDeleted Entries to delete
      */
-    public synchronized void removeEntry(BibEntry toBeDeleted) {
-        removeEntry(toBeDeleted, EntryEventSource.LOCAL);
+    public synchronized void removeEntries(List<BibEntry> toBeDeleted) {
+        removeEntries(toBeDeleted, EntriesEventSource.LOCAL);
     }
 
     /**
-     * Removes the given entry.
-     * The Entry is removed based on the id {@link BibEntry#id}
+     * Removes the given entries.
+     * The entries are removed based on the id {@link BibEntry#id}
      *
      * @param toBeDeleted Entry to delete
      * @param eventSource Source the event is sent from
      */
-    public synchronized void removeEntry(BibEntry toBeDeleted, EntryEventSource eventSource) {
+    public synchronized void removeEntries(List<BibEntry> toBeDeleted, EntriesEventSource eventSource) {
         Objects.requireNonNull(toBeDeleted);
+        //Require not empty?
```

Comment:
I think this is not necessarily. If the list is empty, we are simply doing nothing.